README.md

```diff
@@ -139,6 +139,15 @@ The path can also continue after the sort:
 will sort the friends based on the label of their country,
 and then return their names.
 
+### Accessing collections
+Handle `rdf:List`, `rdf:Bag`, `rdf:Alt`, `rdf:Seq` and `rdf:Container`.
+```javascript
+(async publication => {
+  const authors = await publication['bibo:authorList'].list;
```

Comment:
I would've intuitively
```suggestion
  const authors = await publication['bibo:authorList'].list();
```
but I can't tell you why 🙂 so up to you

Reply:
I agree. However; this doesn't actually work with the current implementation, and will require non-trivial changes to get working (which I have been attempting) due to nested calls to `async` functions.

If there is an easy way of adding this behavior that I am not thinking of please let me know!

Reply:
Actually it should be easy?

Basically you have a `function () { … }` somewhere doing the stuff,
just change that into `function () { return function () { … } }`?

Reply:
Ah of course! I was trying to do this in the wrong place (outside the handler).

Thanks for all the feedback, and Happy Holidays!